app/assets/stylesheets/application.scss

```diff
@@ -24,16 +24,53 @@ body {
   background-size: cover;
 }
 
+footer {
+  background-color: #000000;
+  height: auto;
+}
+
+.banner-image {
+  padding-top: 15px;
+}
+
+.section-title {
+  font-family: 'Miltonian Tattoo', cursive;
+}
+
+.section-box {
+  background-color: #808080;
```

Comment:
Color literals like `#808080` should only be used in variable declarations; they should be referred to via variable everywhere else.